pkgs/tools/filesystems/jfsutils/ar-fix.patch

```diff
@@ -0,0 +1,10 @@
+--- jfsutils-1.1.15/configure.in.orig	2018-11-27 20:46:55.830242385 +0300
++++ jfsutils-1.1.15/configure.in	2018-11-27 20:47:00.596307630 +0300
+@@ -15,6 +15,7 @@
+ AC_PATH_PROG(LN, ln, ln)
+ AC_PROG_LN_S
+ AC_PROG_RANLIB
++AM_PROG_AR
```

Comment:
It would be great if you could send this also to jfsutils maintainer.

Reply:
I still don't get when patch is applied w/ -p0 or -p1. This one required -p1, the reiserfsprogs required a patch to be applied w/ -p0. 

It would be nice if somebody explains it.

Reply:
The `-p` flag (`-p0`, `-p1`, ...) controls how many "levels" are removed from the filenames mentioned in the patch text.

I like the explanation given in BSD's patch: https://www.freebsd.org/cgi/man.cgi?patch(1) 

`-p1` is our default since many patches describe differences in files named `a/foo/bar` vs `b/foo/bar`; with `-p1` we're saying "drop the `a/` and `b/` parts of the filenames used".  `-p0` says don't strip, `-p2` says strip two levels, and so on.
FWIW our `fetchpatch` (built on patchutils) has arguments to adjust patch levels, something I learned much later than I learned about setting `patchFlags` :).  Anyway, hope that helps!